api/utils/sharing/doc.go

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2023, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
General question. What is a consistent year for these headers?

Reply:
Yeah, not sure. This is obviously a copy and paste job ...

Reply:
Not a blocker for this PR. More a general question for follow-up.